runelite-client/src/main/java/net/runelite/client/plugins/inferno/InfernoPlugin.java

```diff
@@ -349,207 +476,638 @@ private void onChatMessage(ChatMessage event)
 		}
```

Comment:
Can you change `indexOf("<")` on line 475 to `indexOf('<')` while you are making changes anyways?